tests/unit/test_charm_certificates_relation_broken.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import tempfile
+from pathlib import Path
+from unittest.mock import Mock
+
+import pytest
+import scenario
+
+from charm import AUSFOperatorCharm
+from lib.charms.tls_certificates_interface.v3.tls_certificates import ProviderCertificate
+from tests.unit.fixtures import AUSFUnitTestFixtures
+
+CONTAINER_NAME = "ausf"
+TEST_POD_IP = b"1.2.3.4"
+TEST_PRIVATE_KEY = b"whatever private key"
+TEST_CSR = b"whatever csr"
+TEST_CERTIFICATE = "whatever certificate"
+TEST_NRF_URL = "https://nrf-example.com:1234"
+
+
+class TestCharmCertificateRelationBroken(AUSFUnitTestFixtures):
+    @pytest.fixture(autouse=True)
+    def context(self):
+        self.ctx = scenario.Context(
+            charm_type=AUSFOperatorCharm,
+        )
```

Comment:
```suggestion
        self.ctx = scenario.Context(
            charm_type=AUSFOperatorCharm,
            capture_deferred_events=True,
            capture_framework_events=True,
        )
```

Reply:
Why do this? We don't leverage this feature

Reply:
To see the all the emitted events, these attributes are used in case we need to  assert on emitted emits.  
This is not needed for this charm . (Trying to indicate the feature that we could use later)